com.unity.netcode.gameobjects/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -640,7 +637,7 @@ private bool GetWriteMethodForParameter(TypeReference paramType, out MethodRefer
                 {
                     var parameters = method.Resolve().Parameters;
 
-                    if (method.Name == k_WriteValueMethodName)
+                    if (method.Name == "WriteValueSafe")
```

Comment:
um, ok...?

Reply:
I am in agreement with Matt what does this have to do with the original PR ?

Reply:
It was an accident. I accidentally did my changes in the wrong project, copy-pasted into testproject without realizing what I was in was out of date, and foolishly didn't look at the diff before submitting. Rollback incoming.